src/components/Language.js

```diff
@@ -30,7 +30,8 @@ export const Language = () => {
     center: i18n('centerkey-select'),
     onKeyCenter,
     left: i18n('softkey-done'),
-    onKeyLeft: () => history.back()
+    onKeyLeft: () => history.back(),
+    onKeyBackspace: () => !query ? history.back() : setQuery(query.slice(0, -1))
```

Comment:
not sure what's the expected behaviour when user press down to hover any item, then press "Back key", 

if press back, delete the latest input char, then your code here is valid; 

but if press back, goes to the previous screen, then we need to change the code here

Reply:
Good call, fined-tuned and revised now